src/mesa.mk

```diff
@@ -42,4 +42,20 @@ define $(PKG)_BUILD
     $(INSTALL) -d '$(PREFIX)/$(TARGET)/lib/mesa'
     $(INSTALL) -m 755 '$(BUILD_DIR)/src/gallium/targets/wgl/libgallium_wgl.dll.a' '$(PREFIX)/$(TARGET)/lib/mesa/'
     $(INSTALL) -m 755 '$(BUILD_DIR)/src/gallium/targets/libgl-gdi/opengl32.dll.a' '$(PREFIX)/$(TARGET)/lib/mesa/'
+
+    # create pkg-config files
```

Comment:
Very nice. Figured it didn't create pkg-config files given a lot of projects "couldn't" find it.